frame/dynamic-fee/Cargo.toml

```diff
@@ -9,10 +9,10 @@ license = "Apache-2.0"
 [dependencies]
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
 serde = { version = "1.0.101", optional = true }
-sp-std = { version = "3.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-core = { version = "3.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-runtime = { version = "3.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-sp-inherents = { version = "3.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
+sp-std = { version = "3.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
+sp-core = { version = "3.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
+sp-runtime = { version = "3.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
+sp-inherents = { version = "3.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
```

Comment:
Those should also enable `std` feature in the below `features` section.

Reply:
There is: https://github.com/PureStake/frontier/blob/d97e1e82fb371bc761f796a852d93b06e487c68a/frame/dynamic-fee/Cargo.toml#L24

I think `default-features = false` was previously missing from these, though the std flag was being forwarded.